Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -2352,6 +2362,17 @@ void WebsiteDataStore::resumeDownload(const DownloadProxy& downloadProxy, const
     networkProcess().send(Messages::NetworkProcess::ResumeDownload(m_sessionID, downloadProxy.downloadID(), resumeData.dataReference(), path, sandboxExtensionHandle, callDownloadDidStart), 0);
 }
 
+HashSet<WebPageProxyIdentifier> WebsiteDataStore::activePages()
+{
+    HashSet<WebPageProxyIdentifier> result;
+    for (auto& pool : WebProcessPool::allProcessPools()) {
+        for (auto& page : pool->pagesUsingWebsiteDataStore(*this))
```

Comment:
We can probably leverage this Hashset function to avoid doing our own loop:
```
template<typename IteratorType>
bool add(IteratorType begin, IteratorType end);
```

Reply:
This is removed.